.travis.yml

```diff
@@ -21,7 +21,7 @@ before_deploy:
 - ./.travis/install_install4j
 - ENGINE_VERSION=$(grep "engine_version" game_engine.properties | sed 's/.*= *//g')
 - TAGGED_VERSION="$ENGINE_VERSION.$TRAVIS_BUILD_NUMBER"
-- ./.travis/build_installers $TAGGED_VERSION
+- TAGGED_VERSION=$TAGGED_VERSION ./gradlew release
```

Comment:
Unfortunately, no.  AFAIK, the child process environment variable definition must appear at the beginning of the command that starts the child process.

I really wish I could find you an "official" reference for this feature for bash or any other Bourne-style shell, but I can't locate one at the moment.  The best I can do is quote you from the bash man page:

> The environment for any simple command or function may be augmented temporarily by prefixing it with parameter assignments, as described above in PARAMETERS.  These assignment statements affect only the environment seen by that command.

This [SE question](https://unix.stackexchange.com/questions/130985/if-processes-inherit-the-parents-environment-why-do-we-need-export) covers this feature, as well as some general theory.

Basically, this line:

```
TAGGED_VERSION="$ENGINE_VERSION.$TRAVIS_BUILD_NUMBER"
```

is defining a _shell variable_ named `TAGGED_VERSION`.  Then this line:

```
TAGGED_VERSION=$TAGGED_VERSION ./gradlew release
```

is creating an _environment variable_ in the child process named `TAGGED_VERSION` whose value is taken from the _shell variable_ `TAGGED_VERSION`.